<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UninstallModal</class>
 <widget class="QDialog" name="UninstallModal">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>375</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>375</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Remove Components</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font-size:14px
}</string>
   </property>
   <property name="text">
    <string>Select the components you would like to remove</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>341</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QCheckBox" name="check4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Full Cleanup (includes manager settings)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Uninstall Spicetify (themes/mods are kept unless option2)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Deactivate themes/mods</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>REMOVE themes/mods</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="bt_apply">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_cancel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="warning">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(0, 0, 0, 0);
font-size: 14px;
font-weight: bold;
</string>
   </property>
   <property name="text">
    <string>This action can NOT be undone!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
